### DISCUSSION TITLE
@tailwindcss/vite not working at all

### DISCUSSION THREAD

**Discussion by cap-mfm** [tailwindlabs/tailwindcss#15465] [Help]
**What version of Tailwind CSS are you using?**
4.0.0-beta.8

**What build tool (or framework if it abstracts the build tool) are you using?**
vite: ^6.0.5

**What version of Node.js are you using?**
v22.12.0

**What browser are you using?**
N.A.

**What operating system are you using?**
macOS

**Reproduction URL**
https://github.com/cap-mfm/tailwind-vite-issue

**Describe your issue**
I've followed the setup guide for v4 beta 1 from the official docs, using the `@taiwindcss/vite` plugin, but no output seems to get generated. The only way to get stuff to work is to install `@tailwindcss/postcss` and use the usual `postcss.config.js` file, even if it's not required in the setups steps when using the vite plugin.

**Comment by wongjn** [comment]
[The version of `@taiwindcss/vite` in the `package.json` seems incorrect, `0.0.0-development.1`](https://github.com/cap-mfm/tailwind-vite-issue/blob/cd594888628151612ea3ebff0ddf6290e681e273/package.json#L16C27-L16C46).

Installing the latest beta version with `npm i @taiwindcss/vite@next` seems to get it working for me.

**Comment by hood** [comment]
alright, i can confirm i accidentally installed `@taiwindcss/vite` not using the `@next` tag. With `@next` everything works as expected, ty
